### PR TITLE
feat: add table controls component

### PR DESCRIPTION
**Pull request by joonchoi0918**
1. Added Controls component for switching table grid/row layout
2. Note: No tests for now

### CODE REVIEW

File: src/components/AsyncTable/Controls/Controls.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+//- Type Imports
```
Comment (domw30) [inline src/components/AsyncTable/Controls/Controls.tsx:1]: nit: we should remove these grouping comments as this is one of the outdated practices from `zns-dapp`

```
import React 'react';
import type { FC } from 'react';

import { IconButton } from '../../IconButton';
import { IconList, IconGrid } from './Icons';
import { SearchBar } from './SearchBar';

import styles from './Controls.module.scss';
```

**Review by domw30 — COMMENTED** [review]
Just the imports in a few of the files on a quick glance Joon. As Brett said we'll keep it local for now and use this at a later date. Thanks!

File: src/components/AsyncTable/Controls/Controls.tsx (hunk 1 above)
Comment (joonchoi0918) [reply src/components/AsyncTable/Controls/Controls.tsx:1]: it is just copied

Comment (joonchoi0918) [reply src/components/AsyncTable/Controls/Controls.tsx:1]: thanks for your comment for this, will keep from now.

Comment (domw30) [reply src/components/AsyncTable/Controls/Controls.tsx:1]: Nice thanks, I forgot to remove them 😄

**Comment by colbr** [comment]
Moved this to #63 - thanks!

**Pull request closed without merging**